portal/static/js/main.js

```diff
@@ -3966,458 +253,228 @@ OrgTool.prototype.populateOrgsList = function(items) {
+            var p = self.getTopLevelParentOrg(o);
+            if (p && !self.inArray(p, parentList)) {
+                parentList.push(p);
             }
-            fillContent.reloadSendPatientEmailForm(userId);
         });
-    });
-};
-OrgTool.prototype.getCommunicationArray = function() {
-    var arrCommunication = [], self = this;
-    $('#userOrgs input:checked').each(function() {
-        if ($(this).val() == 0) return true; //don't count none
-        var oList = self.getOrgsList();
-        var oi = oList[$(this).val()];
-        if (!oi) return true;
-        if (oi.language) {
-            arrCommunication.push({"language": {"coding":[{
-            "code": oi.language,
-            "system": "urn:ietf:bcp:47"
-            }]}});
-        }
-        else if (oi.extension && oi.extension.length > 0) {
-            (oi.extension).forEach(function(ex) {
-                if (ex.url == SYSTEM_IDENTIFIER_ENUM["language"] && ex.valueCodeableConcept.coding) arrCommunication.push({"language": {"coding":ex.valueCodeableConcept.coding}});
-            });
-        };
-    });
-    if (arrCommunication.length == 0) {
-        var defaultLocale = $("#sys_default_locale").val();
-        if (hasValue(defaultLocale)) arrCommunication.push({"language": {"coding":[{
-            "code": defaultLocale,
-            "display":$("#locale").find("option[value='" + defaultLocale + "']").text(),
-            "system": "urn:ietf:bcp:47"
-        }]}});
-
-    };
-    return arrCommunication;
-};
-OrgTool.prototype.getUserTopLevelParentOrgs = function(uo) {
-  var parentList = [], self = this;
-  if (uo) {
-    uo.forEach(function(o) {
-      var p = self.getTopLevelParentOrg(o);
-      if (p && !self.inArray(p, parentList))  {
-        parentList.push(p);
-      };
-    });
-    return parentList;
-  } else return false;
+        uo.parentList = parentList;
+        return parentList;
+    } else return false;
 };
 OrgTool.prototype.getTopLevelParentOrg = function(currentOrg) {
-  if (!currentOrg) return false;
-  var ml = this.getOrgsList(), self = this;
-  if (ml && ml[currentOrg]) {
-    if (ml[currentOrg].isTopLevel) {
-      return currentOrg;
-    } else {
-      if (ml[currentOrg].parentOrgId) return self.getTopLevelParentOrg(ml[currentOrg].parentOrgId);
-      else return currentOrg;
-    };
-  } else return false;
+    if (!currentOrg) {
+        return false;
+    }
+    var ml = this.getOrgsList(), self = this;
+    if (ml && ml[currentOrg]) {
+        if (ml[currentOrg].isTopLevel) {
+            return currentOrg;
+        } else {
+            if (ml[currentOrg].parentOrgId) return self.getTopLevelParentOrg(ml[currentOrg].parentOrgId);
+            else return currentOrg;
+        }
+    } else return false;
 };
 OrgTool.prototype.getChildOrgs = function(orgs, orgList) {
-    if (!orgs || (orgs.length == 0)) {
-      return orgList;
+    if (!orgs || (orgs.length === 0)) {
+        return orgList;
     } else {
```

Comment:
I don't think you need this `else`, since the `if` block exits early, this code will never be reached